### PR TITLE
tunnel: Upgrade WireGuard to version 0.0.36 and wintun to version 0.8

### PR DESCRIPTION
**Pull request by connected-nkozul**
Upgrade WireGuard to version 0.0.36 and Wintun to version 0.8.

### CODE REVIEW

File: tunnel/go.sum [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
 golang.org/x/text v0.3.2/go.mod h1:bEr9sfX3Q8Zfm5fL9x+3itogRgK3+ptLWKqgva+5dAk=
 golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e/go.mod h1:n7NCudcB/nEzxVGmLbDWY5pfWTLqBcC2KZ6jyYvM4mQ=
 golang.zx2c4.com/wireguard v0.0.20191013-0.20191128101113-ddfad453cf22 h1:I+PVPt4NrWyrzoxxgZbXdMalibOrpXTNpqmLsirzSLk=
 golang.zx2c4.com/wireguard v0.0.20191013-0.20191128101113-ddfad453cf22/go.mod h1:P2HsVp8SKwZEufsnezXZA4GRX/T49/HlU7DGuelXsU4=
-golang.zx2c4.com/wireguard/windows v0.0.36-0.20191206131757-3538111295cf h1:pmCAFZj4q79PcaWGtcerro14ZPPXvD7+hK7neYYfP5w=
-golang.zx2c4.com/wireguard/windows v0.0.36-0.20191206131757-3538111295cf/go.mod h1:ge5nAJWkpnWvkzGYUuFWOHoJugGyVpAObDwtkXI/6L4=
+golang.zx2c4.com/wireguard/windows v0.0.36 h1:4Zeux/4ZNFnJ3LEbDouwbrrykgEAGoCkDfwYXZflLfs=
+golang.zx2c4.com/wireguard/windows v0.0.36/go.mod h1:vXbWDbjWCc7no1M/cPEqidiyqYtNOiNoqI5p86wYbTY=
+gopkg.in/Knetic/govaluate.v3 v3.0.0 h1:18mUyIt4ZlRlFZAAfVetz4/rzlJs9yhN+U02F4u1AOc=
+gopkg.in/Knetic/govaluate.v3 v3.0.0/go.mod h1:csKLBORsPbafmSCGTEh3U7Ozmsuq8ZSIlKk1bcqph0E=
```
Comment (zx2c4) [inline tunnel/go.sum]: Remember to run `go mod tidy` to remove these entries.

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://circleci.com/gh/mozilla-services/guardian-vpn-windows/102/artifacts/0/Users/circleci/project/test/result/Output.log' target='_blank'>details</a></strong>

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://circleci.com/gh/mozilla-services/guardian-vpn-windows/104/artifacts/0/Users/circleci/project/test/result/Output.log' target='_blank'>details</a></strong>

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://circleci.com/gh/mozilla-services/guardian-vpn-windows/108/artifacts/0/Users/circleci/project/test/result/Output.log' target='_blank'>details</a></strong>

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://circleci.com/gh/mozilla-services/guardian-vpn-windows/109/artifacts/0/Users/circleci/project/test/result/Output.log' target='_blank'>details</a></strong>

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://circleci.com/gh/mozilla-services/guardian-vpn-windows/112/artifacts/0/Users/circleci/project/test/result/Output.log' target='_blank'>details</a></strong>

**Pull request merged**